lightningd/Dockerfile

```diff
@@ -1,54 +1,24 @@
-ARG BITCOIN_VER=27.2
-FROM blockstream/bitcoind:${BITCOIN_VER} AS builder
+ARG VER=stable
+FROM elementsproject/lightningd:${VER}
 
-# Get CLN
-ARG CLN_VER=v25.05
-ENV CLN_VER=${CLN_VER}
-RUN apt update \
-  && apt install -qfy jq autoconf automake build-essential git libtool libsqlite3-dev libffi-dev \
-                       python3 python3-pip net-tools zlib1g-dev libsodium-dev gettext protobuf-compiler libpq-dev
-RUN pip3 install mako --break-system-packages \
- && pip3 install grpcio-tools --break-system-packages
-RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
-RUN echo 'source $HOME/.cargo/env' >> $HOME/.bashrc
-RUN git clone https://github.com/ElementsProject/lightning.git --depth 5 -b ${CLN_VER} /opt/lightningd
-
-# Build CLN
-WORKDIR /opt/lightningd
-RUN git submodule update --init --recursive --depth 5
-RUN ./configure --prefix=/opt/lightning_install
-RUN make -j$(nproc)
-RUN make install
-
-# Final image
-FROM debian:bookworm-slim
-
-# CLN deps
-RUN apt update \
- && apt install -qfy jq libtool libsqlite3-dev libffi-dev \
-  python3 python3-pip net-tools zlib1g-dev libsodium-dev gettext wget libpq-dev
-
-# Install plugin dependencies (Prom's managed inline with uv)
+# Install Python and required system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends python3 python3-pip python3-venv wget
+# Prepare plugin directory
 ARG PLUGIN_PATH=/opt/plugins
 ARG RAW_GH_PLUGINS=https://raw.githubusercontent.com/lightningd/plugins/master
 RUN mkdir -p ${PLUGIN_PATH}
 
+# Install plugin Python dependencies
 RUN pip3 install --break-system-packages --upgrade pip wheel uv
-RUN pip3 install --break-system-packages -r $RAW_GH_PLUGINS/summary/requirements.txt
+RUN wget -q -O /tmp/requirements.txt $RAW_GH_PLUGINS/summary/requirements.txt && \
+    pip3 install --break-system-packages -r /tmp/requirements.txt
 
-# Add custom plugins (prometheus, summary)
+# Download plugins
 RUN wget -q -O $PLUGIN_PATH/prometheus.py $RAW_GH_PLUGINS/prometheus/prometheus.py
 RUN wget -q -O $PLUGIN_PATH/summary.py $RAW_GH_PLUGINS/summary/summary.py
-RUN chmod a+x $PLUGIN_PATH/*
 RUN wget -q -O $PLUGIN_PATH/summary_avail.py $RAW_GH_PLUGINS/summary/summary_avail.py
+RUN chmod a+x $PLUGIN_PATH/*
```

Comment:
This was intentionally set above the summary_avail.py as that's not a plugin. lightningd picks up everything that's executable as a plugin. Has to be moved back.